frontend/src/styles/global.scss

```diff
@@ -630,6 +630,23 @@ body {
         box-shadow: var(--shadow-elevation);
     }
 
+    .ant-modal-mask {
```

Comment:
I moved this so as I needed to add an override for `.ant-picker-dropdown`, and this is the established cleaner-than-`!important` way of doing Ant styling overrides